firmware/ec/src/filesystem/fs_wrapper.h

```diff
@@ -0,0 +1,25 @@
+/**
+* Copyright (c) 2017-present, Facebook, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree. An additional grant
+* of patent rights can be found in the PATENTS file in the same directory.
+*
+*/
+
+#ifndef SRC_FILESYSTEM_FS_H_
+#define SRC_FILESYSTEM_FS_H_
+
+#include "common/inc/global/post_frame.h"
+
+extern Semaphore_Handle semFilesysMsg;
```

Comment:
alphabetical order

Reply:
As per our current coding guidelines the "#include"s are to be in alphabetical order . If the same needs to be extended for "extern"s as well then we need to change this as part of a separate request as a lot of files might need to be changed .

Reply:
changed as requested